docs/book/src/reference/labels_and_annotations.md

```diff
@@ -1,62 +1,62 @@
+| clusterctl.cluster.x-k8s.io/skip-crd-name-preflight-check        | Can be placed on provider CRDs, so that clusterctl doesn't emit an error if the CRD doesn't comply with Cluster APIs naming scheme. Only CRDs that are referenced by core Cluster API CRDs have to comply with the naming scheme.      | Cluster API | Provider CRDs |
+| controlplane.cluster.x-k8s.io/kubeadm-cluster-configuration      | It is a machine annotation that stores the json-marshalled string of KCP ClusterConfiguration. This annotation is used to detect any changes in ClusterConfiguration and trigger machine rollout in KCP.    | User | Machines |
+| controlplane.cluster.x-k8s.io/remediation-for                    | It is a machine annotation that links a new machine to the unhealthy machine it is replacing.      | User | Machines |
+| controlplane.cluster.x-k8s.io/remediation-in-progress            | It is a KCP annotation that tracks that the system is in between having deleted an unhealthy machine and recreating its replacement.     | User | Machines |
+| controlplane.cluster.x-k8s.io/skip-coredns                       | It explicitly skips reconciling CoreDNS if set.   | User | Control plane machines |
+| controlplane.cluster.x-k8s.io/skip-kube-proxy                    | It explicitly skips reconciling kube-proxy if set.  | User | Control plane machines |
+| machine.cluster.x-k8s.io/certificates-expiry                     | It captures the expiry date of the machine certificates in RFC3339 format. It is used to trigger rollout of control plane machines before certificates expire. It can be set on BootstrapConfig and Machine objects. The value set on Machine object takes precedence. The annotation is only used by control plane machines.      | User | BootstrapConfig and Machine objects |
+| machine.cluster.x-k8s.io/exclude-node-draining                   | It explicitly skips node draining if set.    | User | Machines |
+| machine.cluster.x-k8s.io/exclude-wait-for-node-volume-detach     | It explicitly skips the waiting for node volume detaching if set.      | User | Machines |
+| machinedeployment.clusters.x-k8s.io/desired-replicas             | It is the desired replicas for a machine deployment recorded as an annotation in its machine sets. Helps in separating scaling events from the rollout process and for determining if the new machine set for a deployment is really saturated.   |    Cluster API | Machine deployment's machine sets |
+| machinedeployment.clusters.x-k8s.io/max-replicas                 | It is the maximum replicas a deployment can have at a given point, which is machinedeployment.spec.replicas + maxSurge. Used by the underlying machine sets to estimate their proportions in case the deployment has surge replicas.  |     Cluster API    | Machine deployment's machine sets |
```

Comment:
```suggestion
| machinedeployment.clusters.x-k8s.io/max-replicas                 | It is the maximum replicas a deployment can have at a given point, which is machinedeployment.spec.replicas + maxSurge. Used by the underlying machine sets to estimate their proportions in case the deployment has surge replicas.  |     Cluster API    | MachineSets |
```